.circleci/config.yml

```diff
@@ -238,93 +317,46 @@ jobs:
-            kubectl set image -f kubernetes_deploy/api-sandbox/deployment.yaml cccd-app=${docker_image_tag} --local -o yaml | kubectl apply -f -
-            kubectl apply \
-            -f kubernetes_deploy/api-sandbox/service.yaml \
-            -f kubernetes_deploy/api-sandbox/ingress.yaml
-            kubectl annotate deployments/claim-for-crown-court-defence kubernetes.io/change-cause="$(date +%Y-%m-%dT%H:%M:%S%z) - deploying: $docker_image_tag"
+      - *deploy-sandbox
 
   deploy-staging:
     <<: *cloud_platform_container_config
     steps:
       - checkout
       - setup_remote_docker
-      - run:
-          name: Kubectl deployment setup staging
-          command: |
-            $(aws ecr get-login --region ${AWS_DEFAULT_REGION} --no-include-email)
-            setup-kube-auth
-            kubectl config use-context staging
+      - *setup-staging
       - *decrypt_secrets
-      - deploy:
-          name: Deploy cccd docker image
-          command: |
-            docker_image_tag=${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:${CIRCLE_SHA1}
-            kubectl apply -f kubernetes_deploy/staging/secrets.yaml
-            kubectl set image -f kubernetes_deploy/staging/deployment.yaml cccd-app=${docker_image_tag} --local -o yaml | kubectl apply -f -
-            kubectl apply \
-            -f kubernetes_deploy/staging/service.yaml \
-            -f kubernetes_deploy/staging/ingress.yaml
-            kubectl annotate deployments/claim-for-crown-court-defence kubernetes.io/change-cause="$(date +%Y-%m-%dT%H:%M:%S%z) - deploying: $docker_image_tag"
-
+      - *deploy-staging
+ 
   deploy-production:
     <<: *cloud_platform_container_config
     steps:
       - checkout
       - setup_remote_docker
-      - run:
-          name: Kubectl deployment setup production
-          command: |
-            $(aws ecr get-login --region ${AWS_DEFAULT_REGION} --no-include-email)
-            setup-kube-auth
-            kubectl config use-context production
+      - *setup-prod
       - *decrypt_secrets
-      - deploy:
-          name: Deploy cccd docker image
-          command: |
-            docker_image_tag=${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:${CIRCLE_SHA1}
-            kubectl apply -f kubernetes_deploy/production/secrets.yaml
-            kubectl set image -f kubernetes_deploy/production/deployment.yaml cccd-app=${docker_image_tag} --local -o yaml | kubectl apply -f -
-            kubectl apply \
-            -f kubernetes_deploy/production/service.yaml \
-            -f kubernetes_deploy/production/ingress.yaml
-            kubectl annotate deployments/claim-for-crown-court-defence kubernetes.io/change-cause="$(date +%Y-%m-%dT%H:%M:%S%z) - deploying: $docker_image_tag"
+      - *deploy-prod
 
+  auto-deploy-dev:
+    <<: *cloud_platform_container_config
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *setup-dev
+      - *decrypt_secrets
+      - *deploy-dev
```

Comment:
block is identical to `deploy-dev` so not needed. you can just remove this and call `deploy-dev` below